src/libYARP_wire_rep_utils/WireTwiddler.h

```diff
@@ -167,6 +167,7 @@ class YARP_wire_rep_utils_API WireTwiddlerReader : public yarp::os::InputStream
         pending_string_length = 0;
         pending_string_data = 0;
         override_length = -1;
+        lengthBuffer = yarp::os::NetInt32(0);
```

Comment:
is `lengthBuffer = 0` causing some kind of warning? I don't see the reason to be explicit here